python/ribasim/ribasim/model.py

```diff
@@ -86,6 +88,7 @@ class Model(FileModel):
     user_demand: UserDemand = Field(default_factory=UserDemand)
 
     edge: EdgeTable = Field(default_factory=EdgeTable)
+    crs: str
```

Comment:
Could be below `endtime` attribute, like it is in `config.jl`.

Reply:
I would propose a default here, which could even be the following:

> ENGCRS[\"Undefined Cartesian SRS with unknown unit\", EDATUM[\"Unknown engineering datum\"], CS[Cartesian,2], AXIS[\"X\",unspecified, ORDER[1], LENGTHUNIT[\"unknown\",0]], AXIS[\"Y\",unspecified, ORDER[2], LENGTHUNIT[\"unknown\",0]]]

Reply:
> Could be below endtime attribute, like it is in config.jl.

Done

> I would propose a default here

This has been discussed in the original issue and it was decided to not go for a default.